HW4/VZinkov/HW_4_3.py

```diff
@@ -0,0 +1,13 @@
+a=int(input("Enter your number: "))
+
```

Comment:
n=int(input('enter some number from fibonacci list='))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])
print(f'there are some numbers from fibanacci till that you entered {n}, {fibonacci_numbers}')